jsonschema-validation.xml

```diff
@@ -427,27 +463,39 @@
                 <t>
                     If absent, it can be considered the same as an empty object.
                 </t>
+                <t>
+                    This keyword controls child instance validation.  Validation of the
+                    primitive instance type against this keyword always succeeds.
+                </t>
+                <t>
+                    Child validation succeeds if, for each instance name that matches any
+                    regular expressions that appear as a property name in this keyword's value,
+                    the child instance for that name successfully validates against each
+                    schema that corresponds to a matching regular expression.
+                </t>
             </section>
 
             <section title="additionalProperties">
                 <t>
-                    The value of "additionalProperties" MUST be a boolean or a schema.
+                    The value of "additionalProperties" MUST be a boolean or an
+                    object.  If it is an object, the object MUST be a valid JSON Schema.
```

Comment:
I was about to put up a pull request for #101 which updates all of this a little bit by no longer requiring the boolean case to be specially explained in two places.

I think my wording for that accomplishes your goal, and if not, definitely feel free to fix it in another way.   I'll have the PR up later today.